src/PKSim.Core/Events/EntityEvents.cs

```diff
@@ -155,11 +155,11 @@ public SwapBuildingBlockEvent(IPKSimBuildingBlock oldBuildingBlock, IPKSimBuildi
       }
    }
 
-   public class NoTranporterTemplateAvailableEvent
+   public class NoTransporterTemplateAvailableEvent
    {
       public IndividualTransporter Transporter { get; }
 
-      public NoTranporterTemplateAvailableEvent(IndividualTransporter transporter)
+      public NoTransporterTemplateAvailableEvent(IndividualTransporter transporter)
```

Comment:
This was ideal before because the presenter was already created and we could react to this event. This is not the case before as we instantiate the presenter AFTER . I kept this event but I am not sure how I can use this in an effective way